js/tests/launchAsync.test.ts

```diff
@@ -50,6 +64,26 @@ describe('launchAsync tests', () => {
         }
     });
 
+    it('launches with onExit callback', async () => {
+        jest.useRealTimers();
+        expect.assertions(1);
+
+        const cbOnExit = jest.fn(() => { });
+
+        const options: Options = { onExit: () => { cbOnExit(); } };
+        const launchPromise = launchAsync(SampleToken, SampleSubdomain, SampleContent, options);
+        window.postMessage('ImmersiveReader-LaunchResponse:{"success":true}', '*');
+
+        await launchPromise;
+
+        window.postMessage('ImmersiveReader-Exit', '*');
+
+        // this is to yield this thread of execution to allow the exit message to get processed
+        await new Promise(resolve => { setTimeout(resolve, 1); });
```

Comment:
We can just call setTimeout() in order to have resolve() called after the event loops completes other tasks.

Reply:
can you clarify? 
Do we even need a promise here?